### PR TITLE
Improve user guidance in Kubernetes basics tutorials

### PR DESCRIPTION
**Pull request by adityasamant25**
This PR tries to fix https://github.com/kubernetes/website/issues/43256 

Made the following changes:
1. On the page [Using Minikube to Create a Cluster](https://kubernetes.io/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/ ) removed the link to [Using kubectl to Create a Deployment](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/).
This is done as the [Hello Minikube](https://kubernetes.io/docs/tutorials/hello-minikube/) page already refers to [Using kubectl to Create a Deployment](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/).

2. On the [Hello Minikube](https://kubernetes.io/docs/tutorials/hello-minikube/) page, added a note to highlight that only the minikube installation step from [minikube start](https://minikube.sigs.k8s.io/docs/start/) should be carried out before returning to the steps on this page.

3. Added a short Conclusion to make it clear to the readers that the [Hello Minikube](https://kubernetes.io/docs/tutorials/hello-minikube/) page describes just the basic aspects of the minikube local setup, and now they are in a great position to continue with the rest of the topics.

Suggestions and feedback are welcome.

Best Regards,
Aditya

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@adityasamant25 We do occasionally see Netlify [build failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6570378145af47000856ec57) due to timeouts. Could you try to trigger the build again by pushing an empty commit using `git commit --amend --no-edit && git push --force-with-lease`

**Comment by dipesh-rawat** [comment]
/retitle Improve user guidance in Kubernetes basics tutorials

**Comment by adityasamant25** [comment]
@dipesh-rawat Pushing an empty commit helped pass the build. Thanks for the tip.

File: content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 1]
```diff
@@ -96,9 +96,6 @@ <h2 style="color: #3771e3;">Cluster Diagram</h2>
 
             </div>
         </div>
-        <div class="row">
-            <p>Once you've done that, move on to <a href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">Using <tt>kubectl</tt> to create a Deployment</a>.</p>
-        </div>
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:101]: We could instead remove:
```html
                <p>Now that you know more about what Kubernetes is, visit <a href="/docs/tutorials/hello-minikube/">Hello Minikube</a>
                   to try this out on your computer.</p>
```

and I think I would prefer that.

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

It's a step forward IMO.

**Comment by adityasamant25** [comment]
> **Once this PR has been reviewed and has the lgtm label**, please assign [nate-double-u](https://github.com/nate-double-u) for approval. 

As mentioned above, assigning [nate-double-u](https://github.com/nate-double-u) for approval.

/assign nate-double-u

**Review by Princesso — COMMENTED** [review]
/lgtm

**Review by kumarankit999 — COMMENTED** [review]
Looks Fixed now!!

**Comment by adityasamant25** [comment]
@nate-double-u Could you take a look for approval please?

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**